baton-maven-plugin/src/main/java/org/technologybrewery/baton/AbstractMigration.java

```diff
@@ -51,8 +51,12 @@ public MigrationSummary execute(FileSet[] fileSets) {
                 File fileToMigrate = new File(fileSet.getDirectory(), file);
                 if (shouldExecuteOnFile(fileToMigrate)) {
                     try {
-                        performMigration(fileToMigrate);
```

Comment:
I (Informational): fixed this defect where migrations were reported as successes unless they threw an exception